src/views/RatingsView/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import '../../components/Login/login.scss';
+import { selectedRating } from '../../redux/actions/RatingsActions/actions';
+import RatingComponent from '../../components/Ratings';
+
+class RatingsView extends Component {
+  state = { }
+
+  square = { width: 175, height: 175 }
```

Comment:
What is this?

Reply:
this is a prop that is passed to the Segment component.